ci-operator/step-registry/telco5g/hcp-cluster-setup/telco5g-hcp-cluster-setup-commands.sh

```diff
@@ -292,9 +292,9 @@ cat << EOF > ~/fetch-kubeconfig.yml
     shell: >-
       oc --kubeconfig=/home/kni/.kube/hcp_config_${CLUSTER_NAME} set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=/home/kni/pull-secret.txt
 
-  - name: Patching hostedcluster to disable all default sources
+  - name: Patching hostedcluster to disable all default sources"
```

Comment:
Unexpected changes in this file

Reply:
Good catch, it was changed by make update. I just revert it.